apidoc2swagger/apidoc2swagger.php

```diff
@@ -0,0 +1,160 @@
+<?php
+/**
+ *
```

Comment:
Please add a short-reference to the license this code is in.
Did we already decide on a license?  If not I would propose MIT.
Then you could add a sentence like this to the docblock:
```
This file is part of FileCloud-PHP.
Please check the file LICENSE for information about the license.
```

Here is an example how other open-source projects do this:
https://github.com/symfony/console/blob/master/Terminal.php